src/forms/Form.2.js

```diff
@@ -227,32 +225,37 @@ class Form2 extends Component {
                                   iconText="Where would you like to receive documents related to this case?"
                                   onChange={this.fieldChanged}
```

Comment:
This whole section is redundant code with the Form2DataSection.
I'm working on updating Form2 to use that section